cfg/.coveragerc

```diff
@@ -2,7 +2,7 @@
 
 [report]
 include =
-    */build/*
+    *$TRAVIS_BUILD_DIR*
```

Comment:
Use `/`

Reply:
At the very end? (since it has a trailing slash due to absolute path)